<?xml version="1.0"?>
<launch>
<arg name="model" />

<arg name="gui" default="False" />

<param name="robot_description" command="$(find xacro)/xacro.py '$(find robot_description)/urdf/robot.xacro'"/> 
<!-- <node name="joint_state_publisher" pkg="joint_state_publisher" type ="joint_state_publisher"/> -->

<node name="robot_state_publisher" pkg="robot_state_publisher"
type="state_publisher" >
	<remap from="/joint_states" to="/robot/motor_state"/>
</node>

<node name="motor_state" pkg="move"
                                    type="moveit_motor_state" >
	
</node> 

<node name="rviz" pkg="rviz" type="rviz"/>
</launch>
